### DISCUSSION TITLE
Format failing with "parse info: EOF"

### DISCUSSION THREAD

**Discussion by nick-phillips-dev** [juicedata/juicefs#377] [General]
Running a droplet on DO pointing to a redis instance and Backblaze. I tried both s3 storage option and b2 option with the same result.  

Any ideas?  

```
./juicefs format --storage s3 --bucket https://s3.us-west-002.backblazeb2.com/bucket $REDIS folder
<INFO>: Meta address: redis://doadmin:@private-db-redis
<WARNING>: parse info: EOF
<INFO>: Data uses s3://bucket/folder/
<FATAL>: format: EOF
```

**Comment by chnliyong** [comment]
@hyprnick It seems there's some error when parsing the redis URL, please check the [format doc](https://github.com/juicedata/juicefs#format-a-volume)

**Reply by yuhr123** [reply]
Please note that the documents stucture has changed, please refer to [the new guide.](https://github.com/juicedata/juicefs/blob/main/docs/en/quick_start_guide.md#1-redis-database)

**Comment by xiaogaozi** [comment]
This `redis://doadmin:@private-db-redis` URL looks like should be `redis://:doadmin@private-db-redis`.

**Comment by nick-phillips-dev** [comment]
Thanks all. I’ll give that a shot. 

I tried to connect directly using redis-cli and running into problems because digital ocean’s managed redis requires TLS. 

Does juice support tls for redis?

**Comment by chnliyong** [comment]
> Thanks all. I’ll give that a shot.
> 
> I tried to connect directly using redis-cli and running into problems because digital ocean’s managed redis requires TLS.
> 
> Does juice support tls for redis?

Use scheme `rediss://` for TLS
